docker/test-images/zipkin-cassandra/install.sh

```diff
@@ -184,7 +194,6 @@ function is_cassandra_alive() {
 is_cassandra_alive || exit 1
 
 echo "*** Installing cqlsh"
-# stuck on python2 for compatibility with cassandra 3.x
```

Comment:
I guess this commend doesn't correspond to the command installing and using python 3